osu.Game.Rulesets.Osu/UI/Cursor/GameplayCursor.cs

```diff
@@ -46,28 +54,30 @@ public GameplayCursor()
 
         public bool OnPressed(OsuAction action)
         {
-            switch (action)
-            {
-                case OsuAction.LeftButton:
-                case OsuAction.RightButton:
-                    downCount++;
-                    ActiveCursor.ScaleTo(released_scale).ScaleTo(targetScale, 100, Easing.OutQuad);
-                    break;
-            }
+            if (cursorExpand)
```

Comment:
[nit] you can early return with !cursorExpand
[nit] if with multiline statement usually put in {}